.circleci/config.yml

```diff
@@ -4,14 +4,22 @@ jobs:
     working_directory: ~/tea-shops
     docker:
       - image: circleci/ruby:2.5.3
+
     steps:
       - checkout
       - run:
           name: bundle install
           command: bundle install
+      - run:
+          name: sequel migration test
+          command: sequel -m db/migrations sqlite://tea-shops_test.sqlite
+
+          environment:
+            APP_ENV: 'test'
+
       - run:
           name: sequel migration
-          command: sequel -m db/migrations sqlite://tea-shops.sqlite
+          command: sequel -m db/migrations sqlite://tea-shops_dev.sqlite
```

Comment:
this step should be completely removed, on CI you just need the `test` database.